packages/nimble-components/src/list-option/index.ts

```diff
@@ -66,6 +66,13 @@ export class ListOption extends FoundationListboxOption {
         super.connectedCallback();
         const owner = this.getListOptionOwner();
         owner?.registerOption(this);
+        const observer = new MutationObserver(_ => {
```

Comment:
I'm pretty concerned about creating mutation observers for every option. Can we be more optimal like only for selected options? We are also emitting indiscriminately, I suspect that is a huge number of allocations and firing of junk events.

Reply:
Updated to just observer the selected option.  Regarding emitting indiscriminately, I'll note that we only observe `characterData` changes on the option (and its basically non-existent subtree), and thus, I don't think the events will often fire, specifically for typical usages.

Reply:
I think we should try and frame all performance discussions for "typical usage" in terms of being used as part of virtualized scrolling in the table. In that case I'd expect character data in options to change quickly during scroll

Reply:
Point taken. Even so, I think the only extra optimization I could do is to create a new update tracker that would queue display updates. Not convinced the potential scenario of a a virtualized table would even demand such an optimization.

Reply:
The current PR changes look reasonable